clinvar_ingest/fs.py

```diff
@@ -39,3 +47,17 @@ def find_files(root_directory: str) -> List[str]:
                 filepath = f"{relativized_dir_path}/{filename}"
                 outputs.append(filepath)
     return outputs
+
+
+def _open(filename: str, make_parents=True, mode: BinaryOpenMode = BinaryOpenMode.READ):
+    """
+    Opens a file with path `filename`. If `filename` ends in .gz, opens as gzip.
+
+    If `make_parents` is True, creates parent directories if they do not exist.
+    """
+    if make_parents:
+        for parent in reversed(PurePosixPath(filename).parents):
```

Comment:
Since this is a local path, it should probably use `PurePath`, not `PurePosixPath`, so it'll work on Windows.